public_suffix_list.dat

```diff
@@ -11224,6 +11224,185 @@ info.at
 
 // Michau Enterprises Limited : http://www.co.pl/
 co.pl
```

Comment:
Could you either grant edit access to the maintainers for this PR, or sort this appropriately?

I've validated the _PSL record is in place for all of these domains, and that it matches http://www.co.pl/wp-content/uploads/2014/09/listadomen.txt as the list of domains.